demo/fullstack-app/src/pages/api/readPoints.ts

```diff
@@ -0,0 +1,67 @@
+import { type NextApiRequest, type NextApiResponse } from 'next'
+import { getAuthenticatedDID } from '@ceramic-solutions/key-did'
+import { PointsReader } from '@ceramic-solutions/points'
+import type { ModelInstanceDocument } from '@composedb/types'
+import { type AggregationContent } from '@/utils/types'
+import { fromString } from 'uint8arrays'
+
+const CERAMIC_PRIVATE_KEY: string = process.env.CERAMIC_PRIVATE_KEY ?? ''
+const aggregationModelID: string | undefined = process.env.AGGREGATION_ID ?? undefined
+
+interface Request extends NextApiRequest {
+  body: {
+    recipient: string
+    context: string
+  }
+}
+
+interface Response extends NextApiResponse {
+  status(code: number): Response
+  send(data: { contextTotal: number; total: number } | { error: string }): void
+}
+
+export default async function handler(req: Request, res: Response) {
+  try {
+    const { recipient, context } = req.body
+    //eslint-disable-next-line
+    const seed = fromString(CERAMIC_PRIVATE_KEY, "base16") as Uint8Array;
+
+    // generate issuer for reader context
+    const issuer = await getAuthenticatedDID(seed)
+
+    //create a context reader
+    const contextReader = PointsReader.create({
+      issuer: issuer.id,
+      aggregationModelID,
+    })
+
+    //create a total reader
+    const reader = PointsReader.create({
+      issuer: issuer.id,
+    })
```

Comment:
Same as previous comments, seems like this could be initialized outside of the handler?

Reply:
Much agreed - amended in commit [ac67b90](https://github.com/ceramicstudio/solutions-sdk/pull/18/commits/ac67b907bdd9ed78c7a476a33a8f5630a001b19b)